pennylane/interfaces/jax_jit_tuple.py

```diff
@@ -95,6 +95,18 @@ def _jac_shape_dtype_tuple(tapes, device):
     return tuple(shape_dtypes)
 
 
+def _jit_compute_jvps(jacobians, tangents, multi_measurements, trainable_params):
+    # Remove the the jitted parameters that are not trainable as they are not contributing to the jacobian.
```

Comment:
none of the other interfaces do this, right? I suppose the user wouldn't know the difference (other than jax-jit being fast), but I'm wondering if there are consequences to this

Reply:
nvm, this question doesn't really make sense for other interfaces because this is a jit issue 😄

Reply:
Here the issue is that params are all the jitted one because you need it for the `callback`, but at the same time you want to get the jacobian but only for the trainable ones. Therefore you need to remove them when you get the JVPs. It is only for Jax-jit and only for performance.